app/upgrades/v14_2/upgrades.go

```diff
@@ -33,8 +35,25 @@ func CreateUpgradeHandler(
 			}
 		}
 
+		// Add EIP contained in Shanghai hard fork to the extra EIPs
+		// in the EVM parameters. This enables using the PUSH0 opcode and
+		// thus supports Solidity v0.8.20.
+		logger.Info("adding EIP 3855 to EVM parameters")
+		err := EnableEIPs(ctx, ek, 3855)
+		if err != nil {
+			logger.Error("error while enabling EIPs", "error", err)
+		}
+
 		// Leave modules are as-is to avoid running InitGenesis.
 		logger.Debug("running module migrations ...")
 		return mm.RunMigrations(ctx, configurator, vm)
 	}
 }
+
+// EnableEIPs enables the given EIPs in the EVM parameters.
+func EnableEIPs(ctx sdk.Context, ek *evmkeeper.Keeper, eips ...int64) error {
+	evmParams := ek.GetParams(ctx)
+	evmParams.ExtraEIPs = append(evmParams.ExtraEIPs, eips...)
```

Comment:
Can you sort them? I saw in the test that you made a case where they are not ordered and wondering if it has an effect

Reply:
It doesn't have an effect. That's why I checked the unordered case, to see if there is an error raised on an unordered slice. 

In the interpreter instantiation, it just iterates over all given extra EIPs and calls the individual activator functions. Those just add an entry to a map, so the order doesn't have an effect.

See e.g. for EIP 3855 here: https://github.com/evmos/go-ethereum/blob/v1.10.26-evmos-rc2/core/vm/eips.go#L180-L189